lib-spring/src/main/java/de/muenchen/oss/ezldap/spring/rest/v1/LdapUserController.java

```diff
@@ -179,7 +179,8 @@ public ResponseEntity<List<LdapBaseUserDTO>> searchUser(
                     @ApiResponse(responseCode = "404", description = "OU existiert nicht"),
             }
     )
-    public ResponseEntity<List<LdapBaseUserDTO>> findByOu(@RequestParam(name = "ou") final String ou) {
+    public ResponseEntity<List<LdapBaseUserDTO>> findByOu(@RequestParam(name = "ou") final String ou,
+            @RequestParam(name = "modifyTimeStamp") String modifyTimeStamp) {
```

Comment:
You define RequestParam but then don't use it.

Reply:
That wasn't me. I only removed one parameter that was included in the sources I used.